packages/toolkit/.changes/next-release/Feature-3bebe8f2-b933-4104-8dd6-013d738bd6b5.json

```
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "use ed25519 to generate ec2 ssh key pair"
```

it's usually a good idea to prefix these items with a topic, that makes them easy to read in a list, and also makes it clear which part of the app they refer to. Other notes are [here](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).

```suggestion
	"description": "EC2: use ed25519 to generate ec2 ssh key pair"
```